DigixGlobal/governance-ui-components
# Added subscription for Proposal details Page

[Feature] Auto refreshing of DOM view (Proposal Details Page)

Ref: [DGDG-300](https://tracker.digixdev.com/agiles/88-14/89-17?issue=DGDG-300)

Added subscriptions to Proposal Details page and call to Action Buttons in the Proposal Details Page
- Replaced retrieving of Proposal details from Info-server to Graphql
- Updated Claim Approval Button
- Updated Claim Result Button
- Update Edit Funding Button
- Updated Edit Page

Test Plan:
 - Login to different accounts, Participant and Moderator on a different tab or window
 - Using a Participant account, create a Proposal
 - Ensure you have two accounts logged-in on different windows or tab
 - Using a Moderator account, perform Actions (Approve, Commit, Reveal and etc.)
 - The content, call to action buttons should change as you update or perform an action to the proposal
